dataModel/pom.xml

```diff
@@ -35,12 +35,12 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>${jackson.version}</version>
+            <version>${jackson.spark.version}</version>
```

Comment:
Because spring has dependencies on a specific version and so does spark.

These are being submitted into environments containing spark and its dependencies in the classpath. This is to prevent binary incompatibilities